### ISSUE TITLE
compile error and solution

### ISSUE THREAD

**Issue by colindaven** [bcgsc/ntEdit#2]
This change in the makefile worked for me - Ubuntu 16.04
g++ --version
g++ (Ubuntu 5.4.0-6ubuntu1~16.04.11) 5.4.0 20160609

```
cat Makefile 

#CPPFLAGS= -O3 -std=c++11 -fopenmp -lz 
CPPFLAGS= -O3 -std=c++11 -fopenmp 

all: ntedit

ntedit:
	g++ $(CPPFLAGS) -o ntedit ntedit.cpp -lz

clean: 
	rm ntedit

```

**Comment by warrenlr** [comment]
thank you for reporting both

**Comment by lcoombe** [comment]
This issue was fixed by 1e53e6c050cf3916ae1995ce4e93293183436b74
